app.R

```diff
@@ -548,13 +548,19 @@ ui <- fluidPage(
     ),
     
     # about ----
-    tabPanel(
-      title = div("About", class="about"),
-      # NOTE: when making changes to about.Rmd, move result to www
-      htmltools::tags$iframe(src = "about.html", # put testdoc.html to /www
-                             class="about-panel",
-                             frameborder = 0, 
-                             scrolling = 'auto')
+    navbarMenu("About",
+               tabPanel(
+                 title = div("About", class="about"),
+                 # NOTE: when making changes to about.Rmd, move result to www
+                 htmltools::tags$iframe(src = "www/about.html", # put testdoc.html to /www
+                                        class="about-panel",
+                                        frameborder = 0, 
+                                        scrolling = 'auto'),
+                        HTML(
+                          "Acknowledgements: MIP team, Larke Huang, etc.<br>"),
+                        HTML("Contact: Emilie Gilde (egilde@mitre.org)")),
+               tabPanel("Measure & Methodology Documentation Download", 
+                        includeHTML("www/docdownload.html"))
```

Comment:
Include this in the same way as the above about page.

Reply:
got it. Let me know if I did this correctly, as I was a bit unsure about the class arguments within the iframe and div tags.

Reply:
Fixed this -- src uses www as the place it looks in by default.